EasyPost/Services/BatchService.cs

```diff
@@ -103,13 +106,163 @@ public async Task<Batch> CreateAndBuy(BetaFeatures.Parameters.Batches.Create par
 
         // TODO: Add GetNextPage function when Batches are sorted newest to oldest.
 
+        /// <summary>
+        ///     Add shipments to this batch.
+        /// </summary>
+        /// <param name="parameters">Update shipment parameters.</param>
+        /// <returns>The updated Batch.</returns>
+        [CrudOperations.Update]
+        public async Task<Batch> AddShipments(string id, Dictionary<string, object> parameters)
```

Comment:
issue: we need to be addressing all these Roslyn analysis checks. Please add the missing ID to the docstring.

Reply:
Going to do docstrings holistically in a separate PR